test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json

```diff
@@ -26,10 +26,12 @@
         "dotnet-test-xunit": "1.0.0-dev-*"
       }
     },
-    "dnx451": {
+    "net451": {
+      "frameworkAssemblies": {
+        "System.Threading.Tasks": ""
```

Comment:
It's a test project, so it shouldn't matter. Plus I think `System.Threading.Tasks` is a real fx assembly so it's fine to list it unlike `System.Runtime`.